### PR TITLE
Revamped APIs: Register, Login, ResetPasswordEmail, ResetPassword, ResetPasswordInput, SettingNewPassword, VerifyAccount

### PR DESCRIPTION
**Pull request by lalithbachimanchi**
Please review @HarshadKavathiya @Akhil2096 @Roja1912 @suresh

### CODE REVIEW

File: application/common/constants.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    PASSWORD_CHANGE = "Password was successfully changed"
+    USER_VERIFIED = "User Verification is completed"
+    INVALID_PASSWORD = "The Old password you have entered is Invalid"
+    NEW_TOKEN = "Access Token is generated"
+
+
+class TimeOuts:
+    """Timeouts to be referenced in the code."""
+
+    TWO_FORTY = 240
```
Comment (HarshadKavathiya) [inline application/common/constants.py]: Can we make this variable name like some time in second i.e. for 5 min ==> five_min_in_sec = 300

File: application/api/login.py [hunk 2]
```diff
@@ -1,274 +1,285 @@
...
+            parser.add_argument('email', help='This field cannot be blank', required=True)
+            parser.add_argument('password', help='This field cannot be blank', required=True)
             data = parser.parse_args()
+            list_of_args = [arg.name for arg in parser.args]
+            request_data_validation = validate_empty_fields(data, list_of_args)
+            if isinstance(request_data_validation, dict):
+                return api_response(request_data_validation, STATUS_BAD_REQUEST)
             current_user = User.query.filter_by(email=data['email']).first()
-            expires = datetime.timedelta(hours=240)
+            expires = datetime.timedelta(hours=TimeOuts.TWO_FORTY)
```
Comment (HarshadKavathiya) [inline application/api/login.py]: Instead of TWO_FORTY please use the more proper variable name.

File: application/common/constants.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    PASSWORD_CHANGE = "Password was successfully changed"
+    USER_VERIFIED = "User Verification is completed"
+    INVALID_PASSWORD = "The Old password you have entered is Invalid"
+    NEW_TOKEN = "Access Token is generated"
+
+
+class TimeOuts:
+    """Timeouts to be referenced in the code."""
+
+    TWO_FORTY_IN_HOURS = 240
```
Comment (lalithbachimanchi) [inline application/common/constants.py]: Changed the naming convention @HarshadKavathiya

**Review by HarshadKavathiya — APPROVED** [review]
Looks Good

**Pull request merged**